assets/app/widgets/suse-modal/suse-modal.directive.jade

```diff
@@ -0,0 +1,9 @@
+.modal-content
+    .modal-header
+        button.close(type='button', ng-click='dismiss()', aria-label='Close')
+            span(aria-hidden='true') ×
+        h4#suseModalLabel.modal-title {{ error.title }}
```

Comment:
How is that gonna work? You can only show one modal per page (unless you want to show several at the same time, which is probably terrible UI/UX) and in every page that we have, we call something then continue or fail. Is in that fail when we show the modal with the appropiate error/errors aggregated. 


So basically check -> failure -> modal -> recheck and start again. 

I dont see where we could have more than one modal invoked. This way we reuse the same modal to push and show the error that we want to show.